search/search-client-connect/src/test/java/io/camunda/search/connect/os/ContainerizedSearchDBExtension.java

```diff
@@ -32,16 +37,25 @@ public void beforeAll(final ExtensionContext context) throws Exception {
     asyncOsClient = connector.createAsyncClient();
   }
 
+  /**
+   * @see SearchDBExtension#osClient()
+   */
   @Override
   public OpenSearchClient osClient() {
     return osClient;
   }
 
+  /**
+   * @see SearchDBExtension#asyncOsClient()
```

Comment:
🤔 I think we could also use the inherited doc :) https://stackoverflow.com/a/13327668/2165134

Reply:
Good point, done